rosplan_planning_system/src/PlannerInterface/LPGPlannerInterface.cpp

```diff
@@ -64,7 +64,7 @@ namespace KCL_rosplan {
 		if(dit!=std::string::npos) str.replace(dit,6,domain_path);
 		std::size_t pit = str.find("PROBLEM");
 		if(pit!=std::string::npos) str.replace(pit,7,problem_path);
-        std::string updatePlan = "cp "+data_path+"../../rosplan_planning_system/common/lpgplan.SOL"+" "+data_path+"plan.pddl";
+        std::string updatePlan = "cp "+data_path+"lpgplan.SOL"+" "+data_path+"plan.pddl";
```

Comment:
If data_path does not end in "/", then this will create a strange file. I think the best solution to this would be to modify PlannerInterface.cpp and check for, and add the final slash there.

Reply:
got it. will do.